pytorch_blade/benchmark/TorchBench/test_torch_bench_cpu.sh

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# !/bin/bash
+if [ -f $HOME/.cache/proxy_config ]; then
+  source $HOME/.cache/proxy_config
+fi
+script_dir=$(cd $(dirname "$0"); pwd)
+benchmark_repo_dir=$HOME/.cache/torchbenchmark
+# cache benchmark repo
+if [ ! -d $benchmark_repo_dir ]; then
+    git clone -q https://github.com/pai-disc/torchbenchmark.git --recursive $benchmark_repo_dir
+fi
+
+# parse the arguments
+HARDWARE=$1; shift  ## AArch64-yitian, AArch64-g6r, X86-intel
+JOB=$1; shift       ## tiny, partial, full
+FIELDS=("$@")
+
+# setup for torchbenchmark
+pushd $benchmark_repo_dir
+# cache venv in benchmark dir
+if [[ $HARDWARE == AArch64* ]]; then
+    cp -r /opt/venv_disc ./venv
+fi
+python3 -m virtualenv venv --system-site-packages && source venv/bin/activate
+
+# install dependencies
+python3 -m pip install -q -r $script_dir/requirements_$HARDWARE.txt
+
+git pull && git checkout main
+git submodule update --init --recursive --depth 1
+python3 install.py --continue_on_fail
+
+pushd $script_dir # pytorch_blade/benchmark/TorchBench
+ln -s $benchmark_repo_dir torchbenchmark
+
+# benchmark
+# setup benchmark env
+export DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
+       DISC_CPU_LARGE_CONCAT_NUM_OPERANDS=4 \
+       DISC_CPU_ENABLE_EAGER_TRANSPOSE_FUSION=1 \
+       TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
+
+bench_target=$JOB
+
+binding_cores=0
+config_file=blade_cpu_$JOB.yaml
+results=()
+declare -A threads2cores
+threads2cores=([1]="0" [2]="0-1" [4]="0-3" [8]="0-7")
+for threads in $(echo ${!threads2cores[*]})
+do
+    cores=${threads2cores[$threads]}
+    result=eval-$HARDWARE-fp32_$threads
+    results[${#results[*]}]=$result
+    export OMP_NUM_THREADS=$threads GOMP_CPU_AFFINITY=$cores
+    taskset -c $cores python3 torchbenchmark/.github/scripts/run-config.py \
+            -c $config_file -b ./torchbenchmark/ --output-dir .
+    mv eval-cpu-fp32 $result
+done
+
+# results
+date_str=$(date '+%Y%m%d-%H')
+oss_link=https://bladedisc-ci.oss-cn-hongkong.aliyuncs.com
+oss_dir=oss://bladedisc-ci/TorchBench/cpu/$JOB/$HARDWARE/${date_str}
```

Comment:
How about change to bladedisc-ci/TorchBench/$HARDWARE/$JOB/${date_str}
to make backend folders more obvious